src/Symfony/Component/Notifier/Message/NullMessage.php

```diff
@@ -42,4 +44,13 @@ public function getTransport(): ?string
     {
         return $this->decoratedMessage->getTransport() ?? 'null';
     }
+
+    public function getNotification(): ?Notification
```

Comment:
Why is it needed?

Reply:
AFAIK the notification is not exposed, so I added this method in order to access to the Notification’s properties and assert them in tests context.

Reply:
But as `getNotification()` is not part of the interface, it's fragile. `EmailMessage` does not have a `getNotification()` method either. For now, I would remove constraints needing this.

Reply:
removed